ci-operator/step-registry/openstack/gather/openstack-gather-commands.sh

```diff
@@ -56,9 +58,23 @@ collect_bootstrap_logs() {
 			done
 			
 			SSH_PRIV_KEY_PATH="${CLUSTER_PROFILE_DIR}/ssh-privatekey"
-			openshift-install gather bootstrap --key "${SSH_PRIV_KEY_PATH}" "${GATHER_BOOTSTRAP_ARGS[@]}"
-			cp log-bundle-*.tar.gz "${ARTIFACT_DIR}"
-			echo "${FIPS[@]}" | xargs --no-run-if-empty openstack floating ip delete
+			BOOTSTRAP_GATHER_CMD="/tmp/openshift-install gather bootstrap --key $SSH_PRIV_KEY_PATH ${GATHER_BOOTSTRAP_ARGS[@]}"
+			if test -f "${SHARED_DIR}/proxy-conf.sh"
+			then
+				SSH_ARGS="-o ConnectTimeout=10 -o "StrictHostKeyChecking=no" -i $SSH_PRIV_KEY_PATH"
+				SSH_CMD="ssh $SSH_ARGS $BASTION_USER@$BASTION_FIP"
+				SCP_CMD="scp $SSH_ARGS"
+				echo "Moving credentials and openshift binary to Bastion Proxy"
```

Comment:
I still don't understand why it would be necessary to run the bootstrap gather from the bastion, as we're already using the bastion as a proxy :thinking:

Reply:
@mandre From my understanding the installer [uses its ssh client](https://github.com/openshift/installer/blob/master/cmd/openshift-install/gather.go#L165) to gather the logs from all the servers. I don't see any specification of proxy host being defined in the command. I'm not so sure how that would be handled in the installer, maybe specifying a ProxyCommand or the jump host?

In any case, it looks like that is not implemented at the moment and it might be faster to handle it in our step atm, if possible.

Reply:
Oh, of course... it uses SSH and not HTTP, the proxy is not going to work in this case. It would be nice to configure the SSH client to jump through the bastion rather than copying the binaries and logs to and from the bastion but I don't think that would be doable via a simple ssh config, we would have to change [the installer code](https://github.com/openshift/installer/blob/5a9ad4c024091c6658813ab3a04c8deb61d684cc/pkg/gather/ssh/ssh.go#L23).